src/actions.ts

```diff
@@ -1,16 +1,19 @@
-// @flow
+import {KeyboardEvent} from "react";
```

Comment:
Why is prettier not yelling about this?
```suggestion
import { KeyboardEvent } from "react";
```